design-system/react/src/components/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from "../Box";
+import { Icon } from "../Icon";
+
+import type { MenuProps } from "./Menu";
+import { Menu } from "./Menu";
+
+export default {
+  component: Menu,
+  title: "UI/Menu",
+  argTypes: {},
+};
+
+export const Usage = (args: MenuProps) => (
+  <Box css={{ width: "200px" }}>
+    <Menu {...args} autoFocus disabledKeys={["edit"]} aria-label="Actions">
+      <Menu.Item key="settings" textValue="Settings">
```

Comment:
why we need to inform `textValue`? can we remove it if not needed ?

Reply:
It's for accessibility purposes, also this is something React Aria make mandatory, since it has all accessibility features included